src/mcu.cpp

```diff
@@ -1184,6 +1195,17 @@ int main(int argc, char *argv[])
             {
                 audioDeviceIndex = atoi(argv[i] + 3);
             }
+            else if (!strncmp(argv[i], "-ab:", 4))
+            {
+                char* pColon = argv[i] + 3;
+                
+                do {
```

Comment:
any reason to use do while(0) construction?

Reply:
the do {} while(0) construction is so I can break; out of trying to parse values.
Range checks? I suppose checking if atoi() returns 0 is a sane one(letters/symbols passed to atoi). 0 is dangerous situation, you are right.

Also, I prefer if user makes most of range checks himself.
If insane value is passed and application behaves poorly, maybe insane value should not be passed!

Reply:
normal `if`` `else` can be used here though, no?
I'd prefer default value used if bad values are passed instead of crash or hang